src/components/DataDimension/Operator.js

```diff
@@ -0,0 +1,36 @@
+import { useSortable } from '@dnd-kit/sortable'
+import { CSS } from '@dnd-kit/utilities'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styles from './styles/Operator.style.js'
+
+const DraggableOperator = ({ id, label, type, onDoubleClick }) => {
```

Comment:
File name is called `Operator` but the component is called `DraggableOperator`, change one or the other?